tools/depends/target/python27/Makefile

```diff
@@ -14,6 +14,7 @@ endif
 ifeq ($(OS),ios)
   HOSTPLATFORM=_PYTHON_HOST_PLATFORM="darwin"
   LINK_ICONV=-framework CoreFoundation -liconv
+  CFLAGS+=-DDISABLE_PROCESS_CREATION
```

Comment:
if possible, we shouldn't continue our tradition of misusing CFLAGS for defines. This should be CPPFLAGS